test/.gitignore

```diff
@@ -1 +1,2 @@
 !*.zip
+/0/
```

Comment:
@jglick When you run the tests this folder is created. I though it would be interesting to ignore it.